### ISSUE TITLE
NPM Support & mobile bug

### ISSUE THREAD

**Issue by ceehaupt** [tgomilar/paroller.js#16]
Hey, 
first thanks for the awesome plugin. It would be nice if the plugin supports npm. The other thing is, that the horizontal parallax isn't working on mobile.
Cheers.

**Comment by tgomilar** [comment]
@ceehaupt npm support is added: https://www.npmjs.com/package/paroller.js :wink:

**Comment by powerbuoy** [comment]
While it is installable using NPM - it's not possible to load the plugin with any module loader I know of at least.

Here's what I usually use, I'm not sure it covers every loader out there though:

```
(function (factory) {
	'use strict';

	if (typeof module === 'object' && typeof module.exports === 'object') {
		module.exports = factory(require('jquery'));
	}
	else {
		factory(jQuery);
	}
})(function ($) {
	'use strict';

	// Put your plugin here
	$.fn.myPlugin = function () {
		// ...
	};
});
```

**Comment by powerbuoy** [comment]
Create a PR: https://github.com/tgomilar/paroller.js/pull/18

**Comment by tgomilar** [comment]
Solved with Create a PR: #18. Checkout latest release: [https://github.com/tgomilar/paroller.js/releases/tag/v1.3.0](url) 

Thanks to @powerbuoy 😃
